syz-manager/html.go

```diff
@@ -72,6 +75,15 @@ func (mgr *Manager) httpSummary(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (mgr *Manager) httpConfig(w http.ResponseWriter, r *http.Request) {
+	data, err := json.MarshalIndent(mgr.cfg, "", "\t")
+	if err != nil {
+		fmt.Fprintf(w, "Failed to decode json\n")
```

Comment:
Use http.Error function as it also returns and error HTTP status.
Don't start error messages with a capital letter, this code base doesn't use capital letters for errors.
Also print the error, otherwise it won't be actionable.

Reply:
Changed to http.Error w/o capital letter.